lib/legacy.js

```diff
@@ -18,7 +18,6 @@ module.exports = {
     builtin: true,
     es6: false,
   },
-  ecmaFeatures: {},
```

Comment:
it should be `parserOptions.ecmaFeatures`

http://eslint.org/docs/user-guide/migrating-to-4.0.0#config-validation